Rakefile

```diff
@@ -243,21 +247,25 @@ namespace :jsondoc do
     cp "docs/json/home.html", gh_pages + "json", verbose: true
   end
 
-  desc "Assembles google-cloud umbrella package jsondoc, from gems' jsondoc to gh-pages repo in temp dir."
+  desc "Assembles google-cloud and stackdriver umbrella packages jsondoc, from gems' jsondoc to gh-pages repo in temp dir."
   task :umbrella, :tag do |t, args|
```

Comment:
Can you remove the stackdriver additions to `jsondoc:umbrella` from this PR? Aren't these changes [already in master](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/Rakefile#L257)? Does this PR need to be rebased? I'm surprised that GH shows that it can be merged without conflict.